api/catalog/api/views/media_views.py

```diff
@@ -124,6 +128,16 @@ def report(self, request, *_, **__):
         serializer = self.get_serializer(report)
         return Response(data=serializer.data, status=status.HTTP_201_CREATED)
 
+    def thumbnail(self, image_url, request, *_, **__):
+        serializer = self.get_serializer(data=request.query_params)
+        if not serializer.is_valid():
+            raise get_api_exception("Invalid input.", 400)
```

Comment:
Instead of raising our own exception (which will have fewer details) can we just pass `raise_exception` to `is_valid`?

```suggestion
        serializer.is_valid(raise_exception=True)
```

Reply:
Raise exception from serialiser instead of custom